E-Commerce/readme.md

```diff
@@ -46,13 +46,57 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+-select * from products where product_name like '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+-SELECT products.id,products.product_name,product_availability.unit_price,product_availability.supp_id
+FROM products
+INNER JOIN order_items ON order_items.product_id = products.id
+INNER JOIN product_availability ON product_availability.prod_id = products.id
+WHERE product_availability.unit_price > 100;
```

Comment:
CORRECT - Whilst this is not wrong, there is no need to join in the order_items table, as you are returning no details from it, and do not need it to access the product_availability table.  Also, consider using aliases to save yourself time and to make the query neater (e.g. `SELECT * FROM products p INNER JOIN product_availability pa ON p.id = pa.prod_id`)